salt/spm/__init__.py

```diff
@@ -359,6 +359,7 @@ def _install(self, args):
 
         # First we download everything, then we install
         for package in dl_list:
```

Comment:
Why are there two identical `for` loops here at the same level? Can these be consolidated?